packages/web/src/pages/audio-rewards-page/store/sagas.ts

```diff
@@ -230,19 +230,20 @@ function* claimChallengeRewardAsync(
       challenge_id: challengeId,
       specifier
     }))
+
     const response: { error?: string } = yield* call(
       AudiusBackend.submitAndEvaluateAttestations,
       {
         challenges,
         userId: currentUser.user_id,
         handle: currentUser.handle,
-        recipientEthAddress: currentUser.wallet,
+        recipientEthAddress: currentUser.wallet as string,
```

Comment:
just checking we dont want an if statement check before this in case wallet, endpoints, and parallelization are not around?

Reply:
Yeah actually I think I can add it to the null check above